docs/artifacts_test.md

```diff
@@ -0,0 +1,56 @@
+# Jenkins pipelines
+
+Artifacts tests triggered by another projects which build .rpm's and .deb's. E.g., tests for CentOS triggered by `centos-rpm` project.
+
+There are files for Jenkins in the repository:
+
```

Comment:
I just added a wiki page: https://github.com/scylladb/scylla-artifact-tests/wiki/Migrate-scylla-artifact-tests-to-SCT

It tracked some todo items, some function in original artifact-tests are still not supported in this PR.